kyo-scheduler/jvm/src/main/scala/kyo/scheduler/Queue.scala

```diff
@@ -61,7 +62,7 @@ final private[kyo] class Queue[T](using ord: Ordering[T]) extends AtomicBoolean:
             !isEmpty() && to.isEmpty() && to.tryModify {
                 t = queue.dequeue()
                 val s = size() - 1
-                var i = s - (s / 2)
+                var i = s - Math.ceil(s.toDouble / 2).intValue()
```

Comment:
When a worker had 3 tasks, this code was stealing 2 for example. This new logic makes it steal only 1. Leaving a worker with only one task increases the likelihood of it'll soon go to sleep again. If a thief is successful, it'll naturally try stealing again when it's out of tasks.